tools/RemoveCodeSignArtifacts.ps1

```diff
@@ -0,0 +1,32 @@
+param(
+    [Parameter(Position=0)]
+    [string]$Path
+)
+
+if ($Path -eq $null) {
+    $Path=$PSScriptRoot
+} else {
+    $Path = Resolve-Path $Path
+}
+
+Write-Output "Under the'$Path' folder"
+
+"Signed","Unsigned" | ForEach-Object {
```

Comment:
If this script is basically just removing 2 folders, please do this instead in the build.proj:
```xml
<Exec Command="$(PowerShellCoreCommandPrefix) &quot;Remove-Item -Path $(SignedDirectory),$(UnsignedDirectory) -Recurse -Force -Confirm:$false -ErrorAction Ignore&quot;" ContinueOnError="WarnAndContinue" IgnoreExitCode="true" />
```

Then, we don't need to maintain yet another script.